components/sbm-recipes-mule-to-boot/src/main/java/org/springframework/sbm/mule/actions/javadsl/translators/amqp/AmqConnectorTranslator.java

```diff
@@ -42,6 +42,8 @@ public DslSnippet translate(int id, AmqpConnectorType component,
                                 QName name,
                                 MuleConfigurations muleConfigurations,
                                 String flowName, Map<Class, MuleComponentToSpringIntegrationDslTranslator> translatorsMap) {
-        return new DslSnippet("// FIXME: <amq:connector/>  cannot be translated yet.", Collections.emptySet(), Set.of(), Collections.emptySet());
+        return DslSnippet.builder()
+                .renderedSnippet("// FIXME: <amq:connector/>  cannot be translated yet.")
```

Comment:
Not sure if this applies here, but if this comment is rendered inside a method block it should have a leading space to trigger formatting within OpenRewrite and have the same indent as the statements in the method block.